### DISCUSSION TITLE
Followed docker guide successfully but the subdomain I configured for the guide is unreachable

### DISCUSSION THREAD

**Discussion by Olive-harobed** [chaskiq/chaskiq#977] [General]
Followed docker guide successfully but the subdomain I configured for the guide is unreachable

the domain which the subdomain belongs to has an acm issued wild card certificate, I dont know if to change to https and wss on the .env file, I am also running it on an ec2 instance, the ec2 instance it is running on isnt reachable either

**Comment by michelson** [comment]
Hi, yes, in production for domains with HTTPS you need to set the https://domain wss://domain

**Reply by Olive-harobed** [reply]
Hello, thank you for your response. I took a look at the production variables and was a bit confused about the database url and redis url, it seems I have to set up a separate db and redis from the existing containers used in dev?

Since I am really just testing it out for now and just want to see if it works I opted for continuing with the dev variables and an http subdomain but it is still unreachable. I am wondering apart from following the docs and ensuring the that the containers are up(which works just fine), what else do I have to get this set up running? p.s first time deploying a ruby app so I didnt follow any major steps apart from following the docs and all 
![image](https://github.com/chaskiq/chaskiq/assets/79452458/c1a70320-a64b-464a-bfa4-5767fad9a05a)
![image](https://github.com/chaskiq/chaskiq/assets/79452458/487835df-12b3-4354-bfa5-c7b77ca108ba)
![image](https://github.com/chaskiq/chaskiq/assets/79452458/ce234742-305c-4941-8935-17bade4adcf9)

**Comment by michelson** [comment]
try to configure with localhost:3000 for local environment

**Comment by Olive-harobed** [comment]
I am not clear on this as I am working on an ec2 instance. My local environment cant possible run this app due to the required storage. I doubt configuring with localhost will work out for me, I did try using the public ip of the ec2 instance instead of localhost and still failed

Is there like a video or article that document possible use cases with ec2 instances? 
![image](https://github.com/chaskiq/chaskiq/assets/79452458/516c6268-ffc1-4b7c-b881-0c819ae9f0ae)
![image](https://github.com/chaskiq/chaskiq/assets/79452458/93817030-d158-440c-bd25-8ec67e4aae14)
![image](https://github.com/chaskiq/chaskiq/assets/79452458/d8a78292-dad8-46f8-b66d-cefd991e9faf)

**Reply by michelson** [reply]
for production you need to use the production build, you can use the chaskiq:chaskiq image
